### PR TITLE
Setup Basic File Structure

### PR DESCRIPTION
**Pull request by seancorc**
* File structure inspriation comes from the [Acai Backend](https://github.com/cuappdev/acai-backend) and [this article](https://scotch.io/tutorials/setting-up-a-node-project-with-typescript)

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
```
Comment (alanna-zhou) [inline tsconfig.json]: can these be alphabetized or is this just the set order you have to do

**Review by alanna-zhou — APPROVED** [review]
very cool sean

**Review by orkosinha — APPROVED** [review]
lgtm, you should be able to alphabetize  `tsconfig.json` without any issues like Alanna said

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "coffee-chats-backend",
+  "version": "1.0.0",
+  "description": "backend for coffee chats app",
```
Comment (leewang-codes) [inline package.json]: nitpick but let's capitalize where appropriate: "Backend for Coffee Chats app"

**Review by leewang-codes — APPROVED** [review]
lgtm!

**Pull request merged**